src/rrs/init/wait.py

```diff
@@ -35,12 +35,13 @@
 import base64
 import time
 from typing import TypedDict
-
 import yaml
```

Comment:
Just how there is a blank line separating the internal modules from the external modules, it is also customary to separate the built-in modules from the non-built-ins. That's the the blank line was here.
```suggestion

import yaml
```